### PR TITLE
Added verbosity to the full instructions displayed in the SMGT web app

### PR DESCRIPTION
**Pull request by isaak-willett**
# Overview
This PR adds verbosity to the our existing full instructions pane in the ground truth web app. It addresses all the points laid out in [77](https://app.zenhub.com/workspaces/allensdk-10-5c17f74db59cfb36f158db8c/issues/alleninstitute/segmentation-labeling-app/77
). These instructions will help users to understand the tools and their usages and provide some error resolutions for commonly seen errors.

# Added 
The following instructions to the ground truth app.
* What browsers we support
* Trace feature instructions
* Movie and image feature instructions
* Common errors and resolution steps

# Validation

### Screenshot of Instructions on Localhost
image
image

### Ground Truth job Screen and Link

image
image

[Ground Truth Example Job](https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fuk2ac5ikj9.labeling.us-west-2.sagemaker.aws%2F&data=02%7C01%7C%7C92306f014b804907403a08d7ed4f965d%7C32669cd6737f4b398bddd6951120d3fc%7C0%7C1%7C637238798395243851&sdata=RutdM%2FnhliTK4i8vIM271mBNrvUP80GJTC5n4uIfwGM%3D&reserved=0)

### CODE REVIEW

File: slapp/app/index.liquid.html [hunk 1]
```diff
@@ -210,11 +210,71 @@
...
+        <h5>Objective</h5>
         <p>Decide whether the region of interest is a cell or not.</p>
-        <div>
-           <p><strong>Example: </strong></p>
-           <p><strong>Response: </strong></p>
-        </div>
+        <br>
+        <h5>Supported Browsers</h5>
+        <p>We currently only support modern Chrome and modern Firefox.
+          Support for other browsers will not be provided, use at your own
```
Comment (kschelonka) [inline slapp/app/index.liquid.html:218]: I don't think things will even show up in Safari/Edge, so let's just remove the "use at your own risk" phrase.

**Comment by kschelonka** [comment]
In general LGTM. Given that our user testers were skipping the instructions, I am concerned that they won't read something of this length. Could you please add the click to enlarge interaction near the beginning of the short instructions (since that's the least obvious interaction, as we don't have any icon, etc.)? I like the wording you used.

File: slapp/app/index.liquid.html (hunk 1 above)
Comment (isaak-willett) [reply slapp/app/index.liquid.html]: Removed the phrase!

**Comment by isaak-willett** [comment]
> In general LGTM. Given that our user testers were skipping the instructions, I am concerned that most won't read something of this length. Could you please add the click to enlarge interaction near the beginning of the short instructions (since that's the least obvious interaction, as we don't have any icon, etc.)? I like the wording you used.

Moved it to the top of the list of instructions.

**Comment by kschelonka** [comment]
> Moved it to the top of the list of instructions.
Thanks. Can you also please add it to the <short-instructions>?

**Comment by isaak-willett** [comment]
> 
> 
> > Moved it to the top of the list of instructions.
> 
> Thanks. Can you also please add it to the <short-instructions>?

I added it to the short instructions and formatted the instructions a little more.
image

**Pull request merged**